frontend/src/js/controllers/ChallengeInviteCtrl.js

```diff
@@ -0,0 +1,71 @@
+// Invoking IIFE
+(function () {
+
+    'use strict';
+
+    angular
+        .module('evalai')
+        .controller('ChallengeInviteCtrl', ChallengeInviteCtrl);
+
+    ChallengeInviteCtrl.$inject = ['utilities', 'loaderService', '$scope', '$rootScope', '$state', '$http', '$stateParams'];
+
+    function ChallengeInviteCtrl(utilities, loaderService, $scope, $rootScope, $state, $http, $stateParams) {
+
+        var vm = this;
+        vm.invitationKey = $stateParams.invitationKey;
+
+        vm.registerChallengeParticipant = function (registerChallengeParticipantForm) {
+            if (registerChallengeParticipantForm) {
```

Comment:
You don't need this check. You can disable the submit button if form is not filled.